DIGNDB.App.SmitteStop.API/Services/RollingKeyPairGeneratorWithLocalCertificate.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DIGNDB.App.SmitteStop.API.Attributes;
+using DIGNDB.App.SmitteStop.API.Services;
+using Org.BouncyCastle.Asn1.X9;
+using Org.BouncyCastle.Crypto.EC;
+using Org.BouncyCastle.Math;
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace DIGNDB.App.SmitteStop.API
+{
+    public class RollingKeyPairGeneratorWithLocalCertificate : RollingKeyPairGenerator, IAnonymousTokenKeySource
+    {
+        public RollingKeyPairGeneratorWithLocalCertificate(string thumbprint)
+            : base(LocateLocalCertificate(thumbprint), CustomNamedCurves.GetByOid(X9ObjectIdentifiers.Prime256v1))
+        {
+        }
+
+        private static X509Certificate2 LocateLocalCertificate(string thumbprint)
+        {
+            using (var store = new X509Store(StoreName.My, StoreLocation.LocalMachine))
```

Comment:
Instead of `using` block you can use
```
using var store = new X509Store(StoreName.My, StoreLocation.LocalMachine);
```
Just a suggestion for future pull requests.

Reply:
Fixed. This is neat. Makes me feel like I'm back in C++ again :-)